File: rest/test-credentials/test-calls-example-2/test-calls-example-2.5.x.php
<?php
// Get the PHP helper library from twilio.com/docs/php/install
require_once '/path/to/vendor/autoload.php'; // Loads the library
use Twilio\Rest\Client;

// Your Account Sid and Auth Token from twilio.com/user/account
$sid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
$token = "your_auth_token";
$client = new Client($sid, $token);

$call = $client->calls->create(
    "+15005550003",
    "+15005550006",
    array("url" => "http://demo.twilio.com/docs/voice.xml")
);

echo $call->sid;

File: rest/test-credentials/test-calls-example-2/test-calls-example-2.5.x.rb
# Get twilio-ruby from twilio.com/docs/ruby/install
require 'twilio-ruby'

# Get your Account SID and Auth Token from twilio.com/console
# To set up environmental variables, see http://twil.io/secure
account_sid = ENV['TWILIO_ACCOUNT_SID']
auth_token = ENV['TWILIO_AUTH_TOKEN']
@client = Twilio::REST::Client.new(account_sid, auth_token)

call = @client.calls.create(url: 'http://demo.twilio.com/docs/voice.xml',
                            to: '+15005550003',
                            from: '+15005550006')
puts call.direction
